ddp-workspace/projects/ddp-dsm-ui/src/app/utils/utils.ts

```diff
@@ -53,7 +53,7 @@ export class Utils {
     if (date != null) {
       return date.toString();
     }
-    return '';
+    return null;
```

Comment:
This method looks like it's called in about 40 places.  Are we sure that they all will handle the change from '' to null?

Reply:
Changed this so it is set to null only for fax values in onc history